HHS/Head-Start-TTADP
# [PROD] Dashboard heading should not change, deploy scripts to update participants and topics

## Description of change

- The dashboard heading no longer updates based on the selected region, the notifications are now entirely within the pills
- Scripts to update participants and topics based on data feedback will be deployed (and executed separately)
- Tech debt/backend only change - old single approver database columns and code have been removed
- A backend only change was made to prepare for the audit log

## How to test

- There are two scenarios: A user has one region, and sees a heading like "Region 1 TTA Activity Dashboard" or a user can view different regions, in which case they'd see "Regional TTA Activity Dashboard". Dashboard should work properly

The other changes won't be visible or they won't be visible until the scripts are executed.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-624
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-581
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-582

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
